src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -31,27 +31,14 @@ public void isSamePerson() {
         // null -> returns false
         assertFalse(ALICE.isSamePerson(null));
 
-        // different phone and email -> returns false
-        Person editedAlice = new PersonBuilder(ALICE).withPhone(VALID_PHONE_BOB).withEmail(VALID_EMAIL_BOB).build();
-        assertFalse(ALICE.isSamePerson(editedAlice));
+        // same name, different attributes -> returns true
+        Person editedAlice = new PersonBuilder(ALICE).withPhone(VALID_PHONE_BOB).withEmail(VALID_EMAIL_BOB)
+                .withAddress(VALID_ADDRESS_BOB).withTags(VALID_TAG_HUSBAND).build();
+        assertTrue(ALICE.isSamePerson(editedAlice));
 
         // different name -> returns false
```

Comment:
Similarly, `// different name -> all other attributes same -> returns false`

Reply:
For consistency with the earlier comment https://github.com/se-edu/addressbook-level3/pull/75#discussion_r550386291, I think it should be this instead? :P

// different name ~~->~~**,** all other attributes same -> returns false